derive/src/lib.rs

```diff
@@ -16,7 +18,10 @@ use std::collections::HashSet;
 use std::error::Error;
 use syn::ext::IdentExt;
 use syn::parse::{Error as ParseError, Parse, ParseStream, Result as ParseResult};
-use syn::{braced, parse_macro_input, Error as SynError, Ident, LitInt, LitStr, Token, Visibility};
+use syn::{
+    braced, parenthesized, parse_macro_input, Error as SynError, Ident, LitInt, LitStr, Token,
+    Visibility,
+};
```

Comment:
I assume this is some sort of change to the linter's line width since there does not appear to be anything new imported here.

Reply:
`Visibility` was added.

Reply:
Ahh yes, my viability isn't what it used to be!